src/main/java/payapal/Perfection.java

```diff
@@ -0,0 +1,42 @@
+package main.java.payapal;
+
+public class Perfection {
+
+    private static Perfection perf;
```

Comment:
When we use Singleton usually the name of the variable holding the Singleton istance is called instance